blackjack/cards.py

```diff
@@ -1,7 +1,7 @@
 """Create Card and Deck classes as well as module variables for card printing"""
 
 from colorama import init
-from blackjack.cards_config import SUIT_SYMBOL_DICT, CARD_FACE_DICT
+from blackjack.cards_config import SUIT_SYMBOL_DICT, CARD_FACE_DICT, COLOR_RESET
```

Comment:
What if instead we used `Style.RESET_ALL` from `colorama`? Did you try that?

Reply:
Pushed an update to this PR with that change.  Works just fine!